From: David Mohammed <fossfreedom@ubuntu.com>
Date: Sun, 23 Apr 2017 00:08:00 +0100
Subject: [PATCH] Disable popover animations

Testing of budgie-desktop on Ubuntu Budgie 17.04 demonstrated
popover animations flickered in the panel.  This is a regression
that was first observed in 10.3.1; so we resolved this by dropping
animations. This patch is to ensure we keep the same popover
animation mechanism as v10.2.9.
---Index: budgie-desktop-10.3.1/src/panel/popovers.vala
===================================================================
--- budgie-desktop-10.3.1.orig/src/panel/popovers.vala
+++ budgie-desktop-10.3.1/src/panel/popovers.vala
@@ -100,11 +100,7 @@ public class PopoverManagerImpl : Popove
 
     void hide_popover()
     {
-        #if HAVE_GTK_322
-        visible_popover.popdown();
-        #else
         visible_popover.hide();
-        #endif
         make_modal(visible_popover, false);
         visible_popover = null;
     }
@@ -162,11 +158,7 @@ public class PopoverManagerImpl : Popove
             }
             Idle.add(()=>{
                 if (!pop.get_visible()) {
-                    #if HAVE_GTK_322
-                    pop.popup();
-                    #else
                     pop.show();
-                    #endif
                 }
                 return false;
             });
